doc/sphinx/source/recipes/recipe_python.rst

```diff
@@ -0,0 +1,61 @@
+.. _recipe_python:
+
+Example recipe Python
+=====================
+
+Overview
+--------
+
+This is an example recipe calling a simple diagnostic script written in Python.
+The recipe produces time series plots of global mean temperature and for the
+temperature in Amsterdam. It also produces a map of global temperature in
+January 2020.
+
+Data are available e.g. via the `climate4impact portal <https://climate4impact.eu/impactportal/data/esgfsearch.jsp>`_. The links below will help you find them:
+
+  * `BCC-ESM1 <https://climate4impact.eu/impactportal/data/esgfsearch.jsp#project=CMIP6&variable=tas&frequency=mon&experiment_id=historical&member_id=r1i1p1f1&source_id=BCC-ESM1&>`_
+
+  * `CanESM2 <https://climate4impact.eu/impactportal/data/esgfsearch.jsp#project=CMIP5&variable=tas&experiment=historical&model=CanESM2&ensemble=r1i1p1&time_frequency=mon&>`_
```

Comment:
It would be better to refer to the chapter on obtaining input data: https://docs.esmvaltool.org/en/latest/input.html. Maybe we could mention the climate4impact portal there, as a modern looking alternative to the standard web interface provided by ESGF nodes.

I do like the idea of just providing some direct download links for the really impatient, but maybe it would be best to not do that via the climate4impact portal, but rather directly link to an esgf node, e.g. http://esgf2.dkrz.de/thredds/fileServer/lta_dataroot/cmip5/output1/CCCma/CanESM2/historical/mon/atmos/Amon/r1i1p1/v20120718/tas/tas_Amon_CanESM2_historical_r1i1p1_185001-200512.nc

Reply:
Yes that's what I had before, but then I was doubting whether we should be doing this. For CMIP6 the license is in the file itself, but for CMIP5 it is not. So I figured that if we provide download links, we should probably also publish that licence info. That's why I referred to an official data portal instead.

Reply:
I'm not sure if climate4impact is an official data portal though, I would refer to an esgf node site then, e.g. https://esgf-data.dkrz.de/projects/cmip5-dkrz/

Reply:
@bouweandela is it okay like this?

Reply:
Looks good now